package.json

```diff
@@ -48,7 +48,7 @@
     "@semantic-release/git": "10.0.1",
     "@typescript-eslint/eslint-plugin": "^4.33.0",
     "@typescript-eslint/parser": "^4.24.0",
-    "axios": "^0.26.1",
+    "axios": "^1.6.5",
```

Comment:
I actually think you can remove this dependency altogether. It's probably not used anywhere.

Reply:
Also you might want to update the `ibm-cloud-sdk-core` package to 4.1.2 while making changes (that's the version guaranteed with v3.80.0 of the generator).